packages/shorebird_code_push/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.1.0+1
```

Comment:
We can probably omit the CHANGELOG for now 🤷

Reply:
Done.